src/components/ErrorBoundary/index.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { ErrorInfo } from 'react'
+import { ExternalLink, ThemedBackground, TYPE } from '../../theme'
+import { AutoColumn } from '../Column'
+import styled from 'styled-components'
+import ReactGA from 'react-ga'
+
+const FallbackWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  align-items: center;
+  z-index: 1;
+`
+
+const BodyWrapper = styled.div<{ margin?: string }>`
+  position: relative;
+  margin-top: 1rem;
+  max-width: 60%;
+  width: 100%;
+`
+
+const CodeBlockWrapper = styled.div`
```

Comment:
what about the <code> tag?

Reply:
that looks a lot better indeed

<img width="585" alt="Screenshot 2021-05-07 at 22 47 08" src="https://user-images.githubusercontent.com/11065661/117506693-f02a2a00-af7d-11eb-8b31-4246d561f46a.png">